sound/soc/sof/ipc.c

```diff
@@ -249,15 +249,15 @@ static int tx_wait_done(struct snd_sof_ipc *ipc, struct snd_sof_ipc_msg *msg,
 				 msecs_to_jiffies(sdev->ipc_timeout));
 
 	if (ret == 0) {
-		dev_err(sdev->dev, "error: ipc timed out for 0x%x size %d\n",
+		dev_err(sdev->dev, "ipc tx timed out for 0x%x size %d\n",
```

Comment:
use '%#x' while we're at it, we started changing this but it's still not consistent.

Reply:
OK, we can do that while we are doing change.